honeybadgermpc/betterpairing.py

```diff
@@ -111,7 +111,7 @@ def __pow__(self, other):
             try:
                 intother = int(other)
                 exponend = ZR(intother)
-            except:
+            except Exception:
```

Comment:
This is too broad - let's figure out what a type error would throw and just catch that